### PR TITLE
Add HA Status Icon

### PR DESCRIPTION
**Pull request by erinix**
NAS-101532

### CODE REVIEW

**Comment by dszidi** [comment]
Could we try this with the same icons that were in the mockups? They are from the same webfont collection that we pulled in for the harddisk icon. The icon names are below...

- lan-connect
- lan-disconnect
- lan-pending

If they don't show up then please let me know. We might need to update the web font.

File: src/app/helptext/topbar.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    ha_status_text_enabled : T('HA Enabled'),
+    ha_is_enabled : T('HA is enabled'),
+    ha_status_text_disabled : T('HA Disabled'),
+    ha_disabled_reasons : {
+        NO_VOLUME : T('No pools are configured'),
+        NO_VIP : T('No interfaces configured with Virtual IP'),
+        NO_SYSTEM_READY : T('Other storage controller has not finished booting.'),
+        NO_PONG : T('Other storage controller cannot be reached.'),
+        NO_FAILOVER : T('Failover is administratively disabled.'),
+        NO_LICENSE: T('Other storage controller has no license'),
```
Comment (Mrt134) [inline src/app/helptext/topbar.ts]:
```suggestion
        NO_LICENSE: T('Other storage controller has no license.'),
```

File: src/app/helptext/topbar.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { T } from '../translate-marker';
+
+export default {
+    ha_status : T('HA Status'),
+    ha_status_text_enabled : T('HA Enabled'),
+    ha_is_enabled : T('HA is enabled'),
+    ha_status_text_disabled : T('HA Disabled'),
+    ha_disabled_reasons : {
+        NO_VOLUME : T('No pools are configured'),
+        NO_VIP : T('No interfaces configured with Virtual IP'),
```
Comment (Mrt134) [inline src/app/helptext/topbar.ts]:
```suggestion
        NO_VIP : T('No interfaces configured with Virtual IP.'),
```

File: src/app/helptext/topbar.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { T } from '../translate-marker';
+
+export default {
+    ha_status : T('HA Status'),
+    ha_status_text_enabled : T('HA Enabled'),
+    ha_is_enabled : T('HA is enabled'),
+    ha_status_text_disabled : T('HA Disabled'),
+    ha_disabled_reasons : {
+        NO_VOLUME : T('No pools are configured'),
```
Comment (Mrt134) [inline src/app/helptext/topbar.ts]:
```suggestion
        NO_VOLUME : T('No pools are configured.'),
```

**Pull request merged**